src/main/java/seedu/address/model/client/Timezone.java

```diff
@@ -17,21 +22,31 @@
 public class Timezone {
 
     public static final String GMT_STRING = "GMT";
-    public static final int SMALLEST_NEGATIVE_OFFSET = 12;
-    public static final int LARGEST_POSITIVE_OFFSET = 14;
 
-    public static final String MESSAGE_CONSTRAINTS =
-            "Timezone should be in the form \"" + GMT_STRING + "+X\" or \"" + GMT_STRING
-            + "-X\" where X is a number.\n" + "Largest offset is " + GMT_STRING
-            + "+" + LARGEST_POSITIVE_OFFSET + " and smallest offset is " + GMT_STRING
-            + "-" + SMALLEST_NEGATIVE_OFFSET;
+    public static final Set<String> VALID_TIMEZONES = new HashSet<>();
+
+    static {
```

Comment:
I think we should standardize (forgot which other file was it) and just remove the static, and add a comment above saying this is some special static init block

Reply:
I only found the one in `TbmManager.java` but that one doesn't seem to be for class-level variables

Reply:
Oh right ok